### PR TITLE
The version one of ISA

### PR DESCRIPTION
**Pull request by ualiangzhang**
Update the version one of ISA: create the ISA API service comparing two equations represented by presentation MathML.

### CODE REVIEW

**Comment by ualiangzhang** [comment]
@adarshp Please let me know if you have any questions or requests. Thanks!

File: skema/isa/lib.py [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+    for edge in graph.get_edges():
+        x, y = edge.obj_dict['points']
+        label = edge.obj_dict['attributes']['label'].replace('"', '')
+        amatrix[int(x)][int(y)] = op_dict[label] if label in op_dict else 7
+
+    return amatrix, node_labels
+
+
+'''
```
Comment (adarshp) [inline skema/isa/lib.py:76]: Can you move the docstrings inside the functions? Also, can you run the Python code through the Black autoformatter (https://black.readthedocs.io/en/stable/)?

**Review by adarshp — CHANGES_REQUESTED** [review]
Thanks @ualiangzhang ! Just a couple of minor changes needed.

File: skema/isa/lib.py (hunk 1 above)
Comment (ualiangzhang) [reply skema/isa/lib.py]: Thanks for letting me know! I have pushed the changes.

**Pull request merged**